sbi.adoc

```diff
@@ -1,3 +1,32 @@
 [[sbi]]
 == SBI Requirements
 
+The supervisor binary interface (SBI) cite: [SBI] specification defines the interface
+between the supervisor mode and the next higher privilege mode. This section
+defines the mandatory SBI version and extensions implemented by the higher
+privilege mode in order to be compatible with this specification.
+
+=== Minimum SBI version
+The SBI implementation must be v1.0 or higher. 
+
+=== Mandatory SBI extensions
+
+The SBI v1.0 allows optional SBI extensions. However, the following optional
+extensions are mandatory for a compliant system. 
+
+* SBI Hart State Management (HSM)
+
+* SBI System Reset (SRST)
+
+* SBI Performance Monitoring (PMU)
```

Comment:
Is this really mandatory to booting and general operation? Can this be retrofitted to existing systems?

Reply:
If you don't consider "performance monitoring" as a part of general operation, we can ignore that.
Yes, SBI PMU extension works on existing systems with limited capability.
i.e. You can run perf stat on hifive unmatched with SBI PMU for all the hardware hpm counters available.